doc/multi-o.md

```diff
@@ -0,0 +1,119 @@
+
+## Unlocking Account and Connecting to Ethereum
+
+The Livepeer node requires you to unlock your Ethereum account and provide connection information for an Ethereum JSON-RPC provider in order to interact with the [Livepeer Protocol](https://github.com/livepeer/protocol), unless the node is in [standalone transcoder]((https://livepeer.readthedocs.io/en/latest/transcoding.html#scaling-transcoding)) mode.
+
+For brevity we'll exclude these flags in the examples and replace them with `<...ETH SETUP...>`. If no Ethereum account is available the Livepeer node will create one for you.
+
+More detailed instructions for connecting to Ethereum can be found [here](https://livepeer.readthedocs.io/en/latest/quickstart.html#connecting-to-an-ethereum-node). 
+
+```
+livepeer \
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEYSTORE_DIR> \
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This setup allows an Orchestrator to use a seperate Ethereum account for redeeming winning tickets on-chain and paying for that transaction while the ticket recipient is still the Orcherstrator's on-chain registered Ethereum address.
+
+The Orchestrator node will still be responsible for calling `reward`*.
+
+\* _currently, only the on-chain registered address can call `reward`_
+
+1. Start the Redeemer 
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -redeemer \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port> \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the orchestrator node
+
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -orchestrator -transcoder \
+    -redeemerAddr <REDEEMER_HTTP_ADDR> \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)>
+```
+
+
+## Blockchain Service Node with (Multiple) Orchestrator(s)
+
+In this setup a node started with the keys for the on-chain registered address will be responsible for all transactions. 
+
+In order to use multiple orchestrators with this setup, the cluster of orchestrator nodes responsible for transcoding would have to be behind a load balancer (e.g. NGINX). The URI of the load balancer will be the on-chain registered Service URI.
+
+1. Start the Blockchain Service Node 
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -redeemer \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port> \
+```
+
+2. Start an orchestrator node
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -orchestrator -transcoder \
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR> \
+    -redeemerAddr <REDEEMER_HTTP_ADDR> \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)>
+```
+
+## Redeemer + RewardService  + Multiple Orchestrators
```

Comment:
Let's remove the extra space after "RewardService".

Reply:
fixed in daacf97